src/Symfony/Component/Security/Core/Tests/Authorization/Voter/AbstractVoterTest.php

```diff
@@ -9,8 +9,9 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\Security\Tests\Core\Authentication\Voter;
+namespace Symfony\Component\Security\Core\Tests\Authorization\Voter;
```

Comment:
I also realized the test was in the wrong spot - it wasn't moved when the other tests were moved. Fixed that.